src/generators/jsx-ast/constants.mjs

```diff
@@ -150,3 +150,5 @@ export const TYPES_WITH_METHOD_SIGNATURES = [
   'method',
   'classMethod',
 ];
+
+export const TRIMMABLE_PADDING_REGEX = /^[\s:]+/;
```

Comment:
Don't we have a specific file for regex constants? Also fine if this stays as is.